### ISSUE TITLE
I cannot reproduce the results on JPEGCompressionAttack, VAE attack, Crop attack, and Rotation attack.

### ISSUE THREAD

**Issue by archmelow** [NarcissusEx/GuardSplat#5]
I tested your code on the Lego dataset (blender) with a message of 16 bits , and this yields bitaccuracy of 54%, 60%, 70%, 60%, which is much lower than your experimental results stated on the paper. I ran the code about 20 times in a row, but the results remain the same.

If there are additional settings needed to achieve your results, can you share your experiments settings with us? I believe reproducibility of published works are very important.

**Comment by 1057939502** [comment]
May I ask if you can achieve the accuracy rate of the watermark without being attacked

**Comment by 1057939502** [comment]
In eval_watermark.ipynb , it can be seen that the author can achieve a 98% bit accuracy, but I have no idea how to obtain this model file ( CLIP_visual+msg_decoder-32.pt) because the code (make_decoder.py) saves a pkl file instead of a pt file. I think the key to achieving a 98% accuracy rate lies in obtaining this pt file, but I have no idea how to get this file.

**Comment by 1057939502** [comment]
How did you achieve a 98% bit accuracy rate?

**Comment by CIMASA** [comment]
Same problem.

I followed the provided commands and trained the msg_decoder for 100 epochs. Then, I applied watermarks on a pretrained 3DGS model (trained on the LLFF dataset). I further experimented with 100, 500, and 1000 epochs of training. However, the evaluation metrics were not very promising in all cases.

Could you please provide your training settings and the decoder.pt you used?

[ITER 1000] Bit Acc : 54.2500 | psnr : 11.5514 | ssim : 0.2411 | lpips : 0.7117 [17/09 13:50:33]
[ITER 1000] recon : 0.87773 | msg : 3.50138 | off : 0.86945 | Bit Acc : 57.0156: 100%|

**Comment by WarrenZZZZL** [comment]
> Same problem.
> 
> I followed the provided commands and trained the msg_decoder for 100 epochs. Then, I applied watermarks on a pretrained 3DGS model (trained on the LLFF dataset). I further experimented with 100, 500, and 1000 epochs of training. However, the evaluation metrics were not very promising in all cases.
> 
> Could you please provide your training settings and the decoder.pt you used?
> 
> [ITER 1000] Bit Acc : 54.2500 | psnr : 11.5514 | ssim : 0.2411 | lpips : 0.7117 [17/09 13:50:33] [ITER 1000] recon : 0.87773 | msg : 3.50138 | off : 0.86945 | Bit Acc : 57.0156: 100%|

same problem
